src/main/resources/script.sql

```diff
@@ -1,32 +1,57 @@
-create table library_schema.books(
+create table books(
     id serial primary key,
     title text not null
 )
 
-create table library_schema.authors(
+create table authors(
     id serial primary key,
     name text not null,
     surname text not null
 )
 
-create table library_schema.users(
+create table users(
      id serial primary key,
      name text not null,
      surname text not null
 )
 
-create table library_schema.users_books(
+create table users_books(
     id serial primary key,
     userId integer,
     bookId integer,
-    foreign key (userId) references library_schema.users(id),
-    foreign key (bookId) references library_schema.books(id)
+    foreign key (userId) references users(id),
+    foreign key (bookId) references books(id)
 )
 
-create table library_schema.authors_books(
-    id serial primary key,
-    authorId integer,
-    bookId integer,
-    foreign key (authorId) references library_schema.authors(id),
-    foreign key (bookId) references library_schema.books(id)
-)
\ No newline at end of file
+create table users_books(
```

Comment:
you have 2 tables with the same name. check it please)

Reply:
done